bin/deploy-iqe-cji.sh

```diff
@@ -40,7 +40,10 @@ main() {
     --env "$iqe_env" \
     --cji-name "$cji_name" \
     $selenium_arg \
-    --namespace "$ns"
+    --namespace "$ns")
+
+    container=$(oc_wrapper get pod $pod -n $ns -o jsonpath="{.status.containerStatuses[0].name}")
+    oc_wrapper logs -n $ns $pod -c $container -f &
```

Comment:
I think that you should kill this process in a trap

Reply:
@gbenhaim tell me what do you think is the best way of doing it:
- Use the trap command with the process in background (see latest update, but it is failing because it doesn't find the process at the end)
- Run the log following command foreground and then the `oc wait` command just after
- Same as above, but running the log following command with a timeout

Reply:
instead of calling the kill command directly in the trap, you can call a function.
In this function you can run the kill command and || it with ":" .

Reply:
@gbenhaim can you check it again?